src/ProofOfConcepts/ACLFilterChips/ConsumerGroupLabel.js

```diff
@@ -10,9 +11,12 @@ class ConsumerGroupLabel extends React.Component {
     return (
       <div class="row">
         <div class="child">
-          <span class="pf-c-label mas-m-solid smaller-label pf-m-green ">
+          <Label
+            className="pf-c-label mas-m-solid smaller-label pf-m-green"
+            isCompact
```

Comment:
I didn't get a chance to note some comments about this component in my last review, but there are a few items that would be good to address here.

1. While you can apply classes to PF components this way, any PF classes should be specified using the props defined for the component, and only custom classes that we create to extend the component would be defined with `className`.
2. Any custom classes we create should ideally follow a similar naming convention as the PF class names. I think [HTML training module](https://www.patternfly.org/v4/training/html) for overriding variables explains the BEM naming scheme. When we create class names, instead of using the "pf-" namespace, we would just use "mas-" so that we can easily identify which classes we've created. 
3. When we create custom modifier classes, it's good to scope them to a specific concern. E.g. "-compact" makes the component smaller, "-green" makes it green. 

Related to points 2 and 3 above, it would be good to address the `smaller-label` class and styles associated with it. This class doesn't align with other class names, but it also applies more styles than the name implies. We probably don't need a "smaller" class if we can use the PF modifier class that comes from the `isCompact`. But this needs some review to identify any other styles that were included for `smaller-label` that might still be needed, and find/define a class name for them to live under (E.g. maybe some belong under `mas-m-solid`).

Reply:
I think I fixed it in accordance to those points

Reply:
Looking at this again, it looks like you missed converting the `pf-m-[color]` modifier classes to using the react component `color` prop. These modifier classes are PF classes, and we'll want to continue using classes that come from PF rather than recreate them as `mas-m-[color]` classes. 

You can see an example of the `color` prop in the first example on this page: https://www.patternfly.org/v4/components/label